diff --git a/MRML/CMakeLists.txt b/MRML/CMakeLists.txt
index f22d314..e6e9a13 100644
--- a/MRML/CMakeLists.txt
+++ b/MRML/CMakeLists.txt
@@ -39,14 +39,14 @@ generate_ros2_nodes(
   "rosbag2_interfaces/srv/GetRate"
   "rosbag2_interfaces/srv/IsPaused"
   "rosbag2_interfaces/srv/Pause"
-  "rosbag2_interfaces/srv/Play"
+  #"rosbag2_interfaces/srv/Play"
   "rosbag2_interfaces/srv/PlayNext"
   "rosbag2_interfaces/srv/Resume"
   "rosbag2_interfaces/srv/Seek"
   "rosbag2_interfaces/srv/SetRate"
   "rosbag2_interfaces/srv/Snapshot"
-  "rosbag2_interfaces/srv/SplitBagfile"
-  "rosbag2_interfaces/srv/Stop"
+  #"rosbag2_interfaces/srv/SplitBagfile"
+  #"rosbag2_interfaces/srv/Stop"
   "rosbag2_interfaces/srv/TogglePaused"

   "moveit_msgs/srv/GetPositionFK"
